remote-os/debian/build_os_docker.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+BASE=$(dirname "$0")
+
+#Check if example directory exists, if so: delete it.
+if [ -d $BASE/../../configs/example ]; then
+    rmdir -v --ignore-fail-on-non-empty $BASE/../../configs/example
+fi
+#Create example directory with subdirectories
+mkdir -p $BASE/../../configs/example/kernels
+mkdir -p $BASE/../../configs/example/initrds
+mkdir -p $BASE/../../configs/example/signing
+#Copy signing keys for example use to directory
+cp -r -v $BASE/../../testitems/signing/* $BASE/../../configs/example/signing
+
+#Build docker image
+sudo docker build -t debos .
+
+#Build DebianOS reproducible via docker container
+sudo docker run --cap-add=SYS_ADMIN --privileged -it -v $(pwd)/../../:/system-transparency/ debos
+
+#Copy kernel and initramfs from tmp directory to example directory
+cp $BASE/out/*vmlinuz $BASE/../../configs/example/kernels/
+cp $BASE/out/*cpio.gz $BASE/../../configs/example/initrds/
```

Comment:
die dateinamen lieber ausschreiben, es sollen ja auch genau diese sein

Reply:
wenn wir das manifest in nem extra script erstellen, dann waere das auch der Ort, wo kernel und initrd in den config ordner copiert werden sollten. 
vllt hier am ende einfach nur ne echo " kernel und initrd created at path/of/out/dir" oder so